app/main/movie_service.py

```diff
@@ -0,0 +1,39 @@
+import os
+import datetime as dt
+import requests
+
+from app import db
+from app.main.models import Movie
+
+# is it ok to import database inside this module?
```

Comment:
It would be better to do this by dependency injection. Because now you are mixing responsibility. Read about S in SOLID

Reply:
Fixed it! I now pass db as an argument when creating this class.